cost_my_chemo_bot/bots/telegram/handlers/lead.py

```diff
@@ -22,7 +22,9 @@ async def save_lead(message: types.Message, state: FSMContext):
         "FIELDS[EMAIL][0][VALUE_TYPE]": "WORK",
         "FIELDS[PHONE][0][VALUE]": state_data.phone_number,
         "FIELDS[PHONE][0][VALUE_TYPE]": "WORK",
-        "FIELDS[COMMENTS]": f"Курс: {state_data.course_name}",
+        "FIELDS[COMMENTS]": f"""
```

Comment:
Maybe we should add dedent? I think it's not very cool transferring and watching tabs and new lines :D

Reply:
bitrix ignores indents, we can skip dedent